guide/src/class/object.md

```diff
@@ -58,7 +58,7 @@ impl Number {
         // format string, followed by any number of parameters which replace the
         // `{}`'s in the format string.
         //
-        // `.0` is how in Rust you get the first value of a tuple struct.
+        //                       👇 Tuple field access in Rust uses a dot
```

Comment:
Haha love this!

Reply:
I saw this in a https://fasterthanli.me blog the other day and liked it. I figured I'd `~~steal~~ ~~copy~~ learn from him :)